controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/ControllerContainerTest.java

```diff
@@ -43,6 +43,9 @@
 
     private final String controllerServicesXml =
             "<jdisc version='1.0'>\n" +
+            "  <config name=\"container.handler.threadpool\">\n" +
+            "    <maxthreads>10</maxthreads>\n" +
```

Comment:
What was this before? It doesn't seem very in context for this change :)

Reply:
True. I was looking into some slow tests and noticed that the default number of threads was used (500) which is overkill for tests. 🙂